pkg/rook/common/logs.go

```diff
@@ -8,11 +8,9 @@ import (
 	"github.com/rook/kubectl-rook-ceph/pkg/k8sutil"
 )
 
-var ValidCephSubSystems = []string{"osd", "mds", "mon", "mgr", "auth"}
```

Comment:
Could we just move this logs.go up a directory? Seems like we don't need a `common` subdirectory. It's preferred to keep specific package names if we can avoid general names like `common` or `util`.

Reply:
moved logs.go file to the `pkg/root` directory.